ultimate_cron.module

```diff
@@ -217,292 +147,7 @@ function ultimate_cron_prepare_job($name, $hook, $job = NULL) {
-        }
-        else {
-          // Looks like drupal_cron_run() was called unexpectedly.
-          \Drupal::logger('ultimate_cron')->warning('drupal_cron_run() called unexpectedly by @file:@line function @function. Running core in compatibility mode.', array(
-            '@function' => $debug['4']['function'],
-            '@line' => $debug['3']['line'],
-            '@file' => $debug['3']['file'],
-          ));
-        }
-
-        ultimate_cron_cron_run_compatibility();
-      }
-      catch (Exception $e) {
-        // Restore the user.
-        $GLOBALS['user'] = $GLOBALS['ultimate_cron_original_user'];
-        drupal_save_session($GLOBALS['ultimate_cron_original_session_saving']);
-        throw $e;
-      }
-      return array();
-    }
-
-    // Looks like drupal_cron_run() was called just the way we like it.
-    ultimate_cron_cron_run();
-    exit;
-  }
-  return array();
-}
-
-/**
- * Plugin clean up cron job.
- *
- * This is a cron callback for cleanup up plugins.
- *
- * @param CronJob $job
- *   The cron job being run.
- * @param array $arguments
- *   (optional) An array that can have one or more of the following elements:
- *   - type: The type of the plugin (settings, scheduler, launcher, logger).
- *   - name: The name of the plugin (queue, crontab, serial, database).
- */
-function ultimate_cron_plugin_cleanup($job, $arguments) {
-  $type = $arguments['type'];
-  $name = $arguments['name'];
-  $plugin = ultimate_cron_plugin_load($type, $name);
-  $plugin->cleanup();
-}
-
-/**
- * Implements hook_cronapi().
- *
- * Adds clean up jobs for plugins.
- */
-function ultimate_cron_cronapi() {
-  $items = array();
-
-  ctools_include('plugins');
-  $plugin_types = ctools_plugin_get_plugin_type_info();
-  foreach ($plugin_types['ultimate_cron'] as $plugin_type => $info) {
-    foreach (ultimate_cron_plugin_load_all($plugin_type) as $name => $plugin) {
-      if ($plugin->isValid() && method_exists($plugin, 'cleanup')) {
-        $items["ultimate_cron_plugin_{$plugin_type}_{$name}_cleanup"] = array(
-          'title' => t('Ultimate Cron @type @title cleanup', array(
-            '@type' => $info['defaults']['static']['title singular proper'],
-            '@title' => $plugin->title,
-          )),
-          'callback' => 'ultimate_cron_plugin_cleanup',
-          'callback arguments' => array(
-            'type' => $plugin_type,
-            'name' => $name,
-          ),
-        );
-      }
-    }
-  }
-
-  return $items;
-}
-
-/**
- * Implements hook_watchdog().
- *
- * Capture watchdog messages and send them to the loggers.
- *
- * @todo: port this.
- */
-function ultimate_cron_watchdog(array $log_entry) {
-  if (class_exists('Drupal\ultimate_cron\Logger\LoggerBase')) {
-    LoggerBase::hook_watchdog($log_entry);
-  }
-}
-
-// ---------- FIXUPS FOR CORE  ----------
-
-/**
- * Implements hook_cron_alter().
- *
- * Add better description to core modules.
- */
-function ultimate_cron_cron_alter(&$hooks) {
```

Comment:
parts of this need to be moved to the new class about creating default jobs. keep + new task "Improve and complete default values for common cron jobs"